src/test/java/com/google/roomies/servlets/ListingsServletTest.java

```diff
@@ -76,6 +88,53 @@ public void setUp() throws Exception {
 
     DatabaseFactory.setDatabaseForTest(db);
     setRequestParameters();
```

Comment:
The values you're setting here are critical to the behavior of individual tests, so they should be set in the test themselves. Generally reserver setUp for configuring things to default values that are required for a test to run at all, not for things that are required for particular behavior.  
And if you still want to use a setter function like this in the individual tests, the params its setting should be passed in as arguments at the callsite so it is clear from reading the test where they come from. Tests should aim to be "obviously correct" - the reader should not have to go hunting around to figure out where values are set.

Reply:
Moved the setting of parameters to be within each method. Didn't use the setter function because I think it doesn't make it much cleaner (would have to specify the constants anyway).